generators/server/templates/src/routes/rootRouter.js

```diff
@@ -1,7 +1,14 @@
 import express from "express";
+<% if(options["authentication"] === "passport") { -%>
+import apiRouter from "./apiRouter.js";
+<% } -%>
 
 const rootRouter = new express.Router();
 
+<% if(options["authentication"] === "passport") { -%>
+rootRouter.use("/api", apiRouter)
+<% } -%>
+
 //place your server-side routes here
 
```

Comment:
Difference for file structure in server/routes compared to interlock

We don't have the same file structure and router mounting as Interlock for our api routers.

Instead, we have the same folders of routes/api/v1, but there is no apiRouter or v1Router. We simply add routers for API endpoints to the v1 folder, and then import them directly for mounting in the rootRouter

Reply:
Good example in Staff GP App

https://github.com/LaunchAcademy/coding-crossroads/blob/master/server/src/routes/rootRouter.js